FeedStoreChallenge/Core Data/CoreDataFeed.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  CoreDataFeed.swift
+//  FeedStoreChallenge
+//
+//  Created by OLEKSANDR ISAIEV on 13.05.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+@objc(CoreDataFeed)
+final class CoreDataFeed: NSManagedObject {
+	@NSManaged public var timestamp: Date
+	@NSManaged public var coreDataFeedImages: NSOrderedSet
```

Comment:
`CoreDataFeed` is `internal`. No need to mark properties as `public`.

`coreDataFeedImages` could be renamed to `images` since the context is clear here. The `CoreDataFeed` class already represents a `feed` so you avoid duplication like `feed.coreDataFeedImages` as `feed.images`.

Please remove `public` access modifiers and rename `coreDataFeedImages` to `images`.

```suggestion
	@NSManaged var timestamp: Date
	@NSManaged var images: NSOrderedSet
```

Reply:
Done